src/main/java/io/github/darkkronicle/kronhud/mixins/MixinInGameHud.java

```diff
@@ -52,15 +49,19 @@ public void renderScoreboard(MatrixStack matrices, ScoreboardObjective objective
 
     @ModifyArgs(method = "render", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/font/TextRenderer;draw(Lnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/text/Text;FFI)I"))
     public void getActionBar(Args args){
+        ActionBarHud hud = (ActionBarHud) KronHUD.hudManager.get(ActionBarHud.ID);
+        if (hud != null && hud.isEnabled()) {
+            try {
+                Text message = args.get(1);
+                int color = args.get(4);
 
-        Text message = args.get(1);
-        int color = args.get(4);
 
-        ActionBarHud hud = (ActionBarHud) KronHUD.hudManager.get(ActionBarHud.ID);
-        if (hud != null && hud.isEnabled()){
-            args.set(1, new LiteralText(""));// Set the arguments to undefined values to inhibit
-            args.set(4, 0);//                           the vanilla action bar rendering
-            hud.setActionBar(message, color);// give us selves the correct values
+                args.set(1, new LiteralText(""));// Set the arguments to undefined values to inhibit
+                args.set(4, 0);//                           the vanilla action bar rendering
+                hud.setActionBar(message, color);// give us selves the correct values
+            } catch (ClassCastException ignored) {
+                hud.setActionBar(new LiteralText("Something went wrong...(Probaly you've run this mod on Quilt)"), -1);
```

Comment:
Maybe the message could be:
"Something went wrong. This is usually caused by running the mod on Quilt."
Even better, you could use a translation.